templates/base/src/styles/themes/base/def/colors.js

```diff
@@ -0,0 +1,112 @@
+// Region Colors
+export default {
```

Comment:
@sunjain1 these will be coming as NPM dependency of component library. Please don't copy the theme etc that planned for later.

If this is breaking the flow then ignore that part of config for now